misp_stix_converter/stix-to-misp.py

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env python3
-
-##########################
-#     WRAPPER SCRIPT     #
-#     DEALS WITH STIX    #
-#     TO MISP            #
-##########################
-
+"""Wrapper script providing STIX to MISP conversion. """
 
 import argparse
 import logging
-import pyaml
-import time
-import sys
 import os
+import sys
+import time
+
+import pyaml
 
-from misp_stix_converter.servers import misp
 from misp_stix_converter.converters.convert import load_stix
+from misp_stix_converter.servers import misp
+
+LOGGER_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+CONFIG_DEFAULT_PATH = "~/.misptostix/misp.login"
+MISP_CONFIG_KEY = "MISP"
+SSL_CONFIG_KEY = "SSL"
+BLAST_FROM_PAST_PENALTY_SECS = 1.0
+args_src = sys.argv
```

Comment:
I mean... for what reason?

Reply:
the filename is bad for testing, as you cannot import it with the hyphens without tricks, but at least offer some testability, where you could import the "script" as module, and depending on test idea, overwrite the args_src module level var and call main with the list (you can always use unittest.mock, but ... or one can directly call parse_args(["some", "list"]) at least with the fix I echo pushed (because I forgot to inject into the parser.parse_args call.